.github/workflows/reusable_benchmarks.yml

```diff
@@ -130,19 +129,21 @@ jobs:
         ZE_AFFINITY_MASK=1
         echo "ZE_AFFINITY_MASK=$ZE_AFFINITY_MASK" >> $GITHUB_ENV
 
-    - name: Run UMF benchmarks (using scripts from UR)
+    - name: Run UMF benchmarks
       id: benchmarks
       working-directory: ${{env.BUILD_DIR}}
       run: >
-        taskset -c ${{ env.CORES }} ${{ github.workspace }}/ur-repo/scripts/benchmarks/main.py
+        taskset -c ${{ env.CORES }} ${{ github.workspace }}/sycl-repo/unified-runtime/scripts/benchmarks/main.py
         ~/bench_workdir_umf
         --umf ${{env.BUILD_DIR}}
+        --compare baseline
         ${{ inputs.upload_report && '--output-html' || '' }}
+        ${{ inputs.pr_no != 0 && '--output-markdown' || '' }}
         ${{ inputs.bench_script_params }}
 
     - name: Print benchmark results
       if: ${{ always() }}
-      run: cat ${{env.BUILD_DIR}}/benchmark_results.md
+      run: cat ${{env.BUILD_DIR}}/benchmark_results.md || true
```

Comment:
IMO it is better to fail the workflow than to silently ignore any errors

Reply:
there's a legit use cases, where we only have html output (and it's pushed to web), so no markdown in such case will be found.

Reply:
If we have HTML output this step should be disabled by the if clause?

Reply:
`inputs.pr_no != 0` is tied to markdown output option, we could use that.

Reply:
Done.